x-pack/plugins/reporting/export_types/csv_from_savedobject/server/lib/generate_csv_search.ts

```diff
@@ -151,21 +103,22 @@ export async function generateCsvSearch(
         },
       };
     }, {});
-  const { timeFieldName: indexPatternTimeField } = indexPatternSavedObject;
   const docValueFields = indexPatternTimeField ? [indexPatternTimeField] : undefined;
+  const buildCsvParams: [IndexPatternSavedObject, SearchSourceQuery, Filter[], ESQueryConfig] = [
```

Comment:
Decided to do this since buildEsQuery takes complicated structures for params, but isn't in Typescript